docs/metal_vrf.md

```diff
@@ -1,10 +1,10 @@
 ## metal vrf
 
-VRF operations : create, get, delete
+VRF-Route  operations : create, get, delete, update
```

Comment:
This doesn't look correct; we should be adding the `routes-*` operations to the list of VRF operations